en/docs/learn/kubernetes-operators/k8s-api-operator/enabling-privatejet-mode-to-deploy-apis.md

```diff
@@ -14,7 +14,7 @@ Follow the steps below to enable PrivateJet mode to deploy APIs as managed APIs
 
 1. Install and setup [Kubernetes API Operator](https://github.com/wso2/k8s-api-operator/blob/master/docs/GettingStarted/quick-start-guide.md#step-3-install-api-operator). 
```

Comment:
@Jayanie shall we fix this as wel. It is pointing to the master branch. Which have the latest information. But with the next release this will be outdated. Hence better to fix it as well.

Reply:
Fixed with c01d4e690ec26ba4a8f74d07421f8d168a98419e